src/clr-core/common/index.ts

```diff
@@ -24,4 +24,4 @@ export * from './mixins/css-helpers';
 export * from './mixins/unique-id';
 export * from './mixins/apply-mixins';
 export * from './interfaces';
-export { styles as baseStyles } from './base/base.element.css';
+export { styles as baseStyles } from './base/base.element.css.js';
```

Comment:
This will fix the issue temporarily for Jest but once we pull Core components into clr-angular it will break again. We should update all the relative import paths in core to have `.js` at the end like this one. This is a fundamental issue with TypeScripts output not including the `.js` file extension. See discussion here https://github.com/microsoft/TypeScript/issues/16577 

You can see here lit-html and lit-element do the same thing to get around this issue. 
https://github.com/Polymer/lit-element/blob/master/src/lit-element.ts#L18 

This would make this a pretty large PR so we can merge this to fix the Jest issue now and I can open a new issue describing the steps to address the rest or we tackle this now in this PR. Either way is fine.